CHANGES.md

```diff
@@ -1,3 +1,6 @@
+Unreleased
```

Comment:
In my opinion it's better not to have it in changelogs that appear on Hackage (in releases) because it looks like our internal implementation detail. E. g. it's not present in the [last release](https://hackage.haskell.org/package/universum-1.8.1.1/changelog). We can instead ask to add this section if it's absent, e. g.
> I added an entry to the [changelog](https://github.com/serokell/universum/tree/master/CHANGES.md) (creating "Unreleased" section if necessary) if my changes are visible to the users

What do you think?

Reply:
I agree, I'd change this in the PR template.